recipe/meta.yaml

```diff
@@ -51,63 +51,28 @@ requirements:
     - tornado
     - kiwisolver
 
+test:
+  imports:
+    - matplotlib
+    - matplotlib.pyplot
+    - matplotlib._image
+    - matplotlib._path
+    - matplotlib._png
+    - matplotlib._tri
+    - matplotlib.backends._backend_agg
+    - matplotlib.ft2font
+    - matplotlib.ttconv
+    - matplotlib.backends._tkagg  # [not win]
+    - mpl_toolkits
+    - pylab
+
 outputs:
   - name: matplotlib-base
-    script: install.sh  # [not win]
```

Comment:
Thanks! I the outputs are still the same, right?

Reply:
Yeah, basically if you name an output the same as the master package, I think it takes in all the rest of the fields.